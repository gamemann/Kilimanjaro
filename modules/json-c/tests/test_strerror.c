#include "strerror_override.h"
#include "strerror_override_private.h"

#include <stdio.h>

int main(int argc, char **argv)
{
	puts(strerror(10000));
	puts(strerror(999));
	return 0;
}
