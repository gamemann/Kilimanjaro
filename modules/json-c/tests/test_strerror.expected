ERRNO=10000
ERRNO=999
